src/pages/availability-panel/index.tsx

```diff
@@ -9,14 +9,19 @@ import updateTasksStatus from '@/helperFunctions/updateTasksStatus';
 import { AVAILABLE } from '@/constants/task-status';
 import { FEATURE } from '@/constants/task-type';
 import { BASE_URL } from '@/constants/url';
+import { useGetIdleMembersQuery } from '@/app/services/membersApi';
 
 const AvailabilityPanel: FC = () => {
-    const [idleMembersList, setIdleMembersList] = useState<string[]>([]);
     const [unAssignedTasks, setUnAssignedTasks] = useState<task[]>([]);
     const [error, setError] = useState<boolean>(false);
     const [isTaskLoading, setIsTaskLoading] = useState<boolean>(true);
-    const [isMemberLoading, setIsMemberLoading] = useState<boolean>(true);
     const [refreshData, setRefreshData] = useState<boolean>(false);
+    const {
+        data: idleMembersList = [],
+        isError: membersError,
```

Comment:
nitpik: the name `membersError` doesn't seem appropriate to describe the value in the variable something like `isIdleMembersError` would have been better